api/v1beta1/disruption_types.go

```diff
@@ -43,11 +43,12 @@ type DisruptionSpec struct {
 	// +nullable
 	AdvancedSelector []metav1.LabelSelectorRequirement `json:"advancedSelector,omitempty"` // advanced label selector
 	// +nullable
-	Filter          *DisruptionFilter `json:"filter,omitempty"`
-	DryRun          bool              `json:"dryRun,omitempty"`          // enable dry-run mode
-	OnInit          bool              `json:"onInit,omitempty"`          // enable disruption on init
-	Unsafemode      *UnsafemodeSpec   `json:"unsafeMode,omitempty"`      // unsafemode spec used to turn off safemode safety nets
-	StaticTargeting bool              `json:"staticTargeting,omitempty"` // enable dynamic targeting and cluster observation
+	Filter               *DisruptionFilter `json:"filter,omitempty"`
+	DryRun               bool              `json:"dryRun,omitempty"`          // enable dry-run mode
+	OnInit               bool              `json:"onInit,omitempty"`          // enable disruption on init
+	Unsafemode           *UnsafemodeSpec   `json:"unsafeMode,omitempty"`      // unsafemode spec used to turn off safemode safety nets
+	SpecifiedEnvironment string            `json:"specifiedEnv,omitempty"`    // optional field limiting where disruptions can ru
```

Comment:
Any reason why we're not using an annotation instead of modifying the spec for this? To me it's what it should be used for.

> You can use Kubernetes annotations to attach arbitrary non-identifying metadata to objects.